tests/Io/SapiHandlerTest.php

```diff
@@ -369,4 +369,23 @@ public function testLogPrintsMessageWithCurrentDateAndTime(): void
         $sapi = new SapiHandler();
         $sapi->log('Hello');
     }
+
+
+    public function testSendResponseSetsMultipleCookieHeaders(): void
+    {
+        if (headers_sent() || !function_exists('xdebug_get_headers')) {
+            $this->markTestSkipped('Test requires running phpunit with --stderr and Xdebug enabled');
+        }
+
+        $_SERVER['SERVER_PROTOCOL'] = 'http/1.1';
+        $sapi = new SapiHandler();
+        $response = new Response(204, ['Set-Cookie' => ['1=1', '2=2']], '{}');
+
+        $sapi->sendResponse($response);
+
+        $previous = ['Content-Type:', 'Content-Length: 2', 'Content-Type:', 'Content-Type:', 'Content-Type:', 'Content-Type:', 'Content-Type:', 'X-Accel-Buffering: no', 'Content-Type:'];
+        $this->assertEquals(array_merge($previous, ['Set-Cookie: 1=1', 'Set-Cookie: 2=2']), xdebug_get_headers());
+    }
```

Comment:
This test is looking good, I think it makes sense to move your test below the `testSendResponseSendsStreamingResponseWithNoHeadersAndBodyFromStreamDataAndNoBufferHeaderForNginxServer` test, they look similar and we like to keep context related tests together :+1: 

Also keep an eye on the amount of empty lines (e.g. current 373, 389 and 390 are not necessary)

Reply:
I moved `testSendResponseSendsStreamingResponseWithNoHeadersAndBodyFromStreamDataAndNoBufferHeaderForNginxServer` up to below `testSendResponseSendsStreamingResponseWithNoHeadersAndBodyFromStreamData` and `testSendResponseSetsMultipleCookieHeaders` up below `testSendResponseClosesStreamingResponseAndSendsResponseWithNoHeadersAndBodyForNoContentRespons` - what do you think?